tasks/main.yml

```diff
@@ -34,8 +34,10 @@
 
 # The rpm-distro element executes 'semanage' during its cleanup phase.
 - name: Ensure diskimage-builder SELinux dependencies are installed
-  yum:
-    name: policycoreutils-python
+  vars:
+    package: "{{ 'policycoreutils-python' if ansible_distribution_major_version | int == 7 else 'python3-policycoreutils' }}"
```

Comment:
can we change this to package_name to avoid confusion with package?

Reply:
Done